src/static/graphkb-env-config.js

```diff
@@ -1,5 +1,3 @@
-// default config.js file, gets replaced at run-time by env-scripts
```

Comment:
My understanding is that updates to that file should usually not be commited, am I wrong?

Reply:
Pinging @elewis2 . I can also add this file to gitignore

Reply:
But we want the file to be in the repo

Reply:
ohh I see so we just have to ignore the changes at run-time to it right?

Reply:
Yep, I guess so.